test/unit_tests/braket/aws/test_aws_device.py

```diff
@@ -238,6 +238,36 @@ def test_gate_model_sim_schema():
 )
 
 
+@pytest.fixture
+def quantum_task(aws_session, s3_destination_folder):
```

Comment:
can you call this parameterized_task?

Reply:
Won't this always fail?

Reply:
This will always for fail for this PR. Changed to parameterized_quantum_task